src/coreclr/gcinfo/arraylist.cpp

```diff
@@ -1,18 +1,42 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+// Interpreter-FIXME: we get an existing implementation of ASSERTE via PCH that isn't usable
+//  from inside the interpreter, so we need to replace it with our own.
+#undef _ASSERTE
+
+#if defined(_DEBUG)
+
+#if !defined(_MSC_VER)
```

Comment:
Have you tried to delete `__cdecl` on all instances of `assertAbort` instead? Is there a reason why it does not work?

Reply:
It looked important and I don't have an x86 setup to test it on, so it seemed more prudent to preserve it. I can try taking it out and see what happens. Do you know if we have test coverage that would determine whether I've broken assertions on x86?

Reply:
If it is going to build and link successfully, I am not worried about it being broken.

Reply:
We do not have explicit tests that verify that asserts work correctly. I am not worried about asserts being broken without somebody noticing quickly.